src/heat_equations_with_ice_iterative_and_variable_onestep.jl

```diff
@@ -78,8 +84,10 @@ function solve_coupler!(cs::Interfacer.CoupledSimulation, max_iters)
         numeric_z_range_atmos = []
         while iter <= max_iters
             @info("Current iter: $(iter)")
-
+            # Update models
             FieldExchanger.step_model_sims!(cs.model_sims, t)
+
+            # Save the z- and t-ranges
```

Comment:
Do these change from one time step to the next or can we expect them to stay constant? If they stay constant, you should be able to move this outside the time loop.